sentences/pl/homeassistant_HassTurnOn.yaml

```diff
@@ -4,27 +4,29 @@ intents:
     data:
       - sentences:
           - (<turn_on> | <turn_on_light>) <name>
+        response: default
       - sentences:
           - <open> <name>
         response: cover
       - sentences:
           - <open> <name> <area>
         response: cover_area
       - sentences:
+          - <open> garaż
```

Comment:
Hmmm, gdy próbuje zrobić ` python3 -m script.intentfest parse --language pl --sentence 'Otwórz garaż'` dostaję taki błąd na twarz: `jinja2.exceptions.UndefinedError: 'dict object' has no attribute 'name'`

Reply:
You are getting the exception because @witold-gren removed the garage fixture from tests few days ago. @witold-gren can we re-enable it? What was the intent here?

Reply:
Yes but as currently is we can not open an area. You can open covers in area, but no the area itself. And here we are trying to do exactly that 😄 Can we just rename `garaż` to `drzwi garażowe` for now just get going with the translations.